src/farajaland/features/employees/scripts/manage-users.ts

```diff
@@ -45,6 +45,8 @@ export function getScope(role: string, environment: string): string[] {
       return setDemoUser(['sysadmin'], environment)
     case 'NATIONAL_SYSTEM_ADMIN':
       return setDemoUser(['sysadmin', 'natlsysadmin'], environment)
+    case 'PERFORMANCE_MANAGEMENT':
```

Comment:
The role should be PERFORMANCE_MANAGER I think.

Reply:
Actually I have just used the value declared here https://github.com/opencrvs/opencrvs-farajaland/blob/a36547a4f124450588975b9bb0e620f492986ef3/src/farajaland/features/employees/scripts/manage-users.ts#L94
It might also have some references in client. Should I update all these?